packages/babel-parser/test/fixtures/typescript/cast/as-const-2/input.js

```diff
@@ -0,0 +1,3 @@
+let e1 = v1 as const;  // Error
+let e2 = (true ? 1 : 0) as const;  // Error
+let e3 = id(1) as const;  // Error
```

Comment:
For tests that contain multiple cases that all should error, they should be split up in multiple files. Otherwise the parser throws on the first line and never even parsers any consequent line.

Reply:
noted. i've split them into separate files